GNUmakefile

```diff
@@ -43,6 +44,9 @@ test:
 integration:
 	go clean -testcache && go test --tags=integration ./...
 
+report:
+	go clean -testcache && go test -json --tags=integration ./... | go-test-report -o  test-report.html -t "Go SDK Tests report"
+
```

Comment:
1. What will be using that report?
2. Cannot we add that directly to `integration` since everything is almost the same?

Reply:
The report is to have the errors in a html file in gcp.
I didn't use integration to be able to keep it for manual testing in github and locally